docs/clients/rgb_led.rst

```diff
@@ -0,0 +1,132 @@
+    }
+
+Matlab
+~~~~~~
+
+To use the RGB LED Interface client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a RgbLedClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RgbLedClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+
+    % Make a RgbLedClient object with obj_id 0
+    RgbInterface = RgbLedClient(’com’,com);
+
+    % Use the RgbLedClient object
+    red = RgbInterface.get(’red’);
+
+Python
+~~~~~~
+
+To use the RGB LED Interface Client in Python, import ``iqmotion`` and create a module that has the RGB LED Interface Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RGB LED Interface Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0, firmware="speed")
+    
+    red = |variable_name|.get("rgb_led", "red") 
+    print(f"Red Intensity: {red}")
+
+.. _rgb_led_message_table:
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 100 | RGB LED Interface
+    .. table:: RGB LED Interface
+        :widths: 8 18 15 10 5 50
+        :class: tight-table
+
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | Sub ID | Short Name    | Access         | Data Type | Unit                | Note                                                                                                                                                      |
+        +========+===============+================+===========+=====================+===========================================================================================================================================================+
+        | 0      | red           | get, set, save | uint8     |                     | The intensity of the red LED [0, 255]                                                                                                                     |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 1      | green         | get, set, save | uint8     |                     | The intensity of the green LED [0, 255]                                                                                                                   |
```

Comment:
Another thing, while it gets specified in update color below, you may want to mention on red, green, and blue that you need to do update color before it will change. I'm sure people will look at the green endpoint without looking at the update color endpoint and get confused